src/resources/hassio-style.html

```diff
@@ -0,0 +1,52 @@
+<dom-module id='hassio-style'>
+  <template>
+    <style>
+      paper-card {
```

Comment:
Using elements as CSS selector in a reusable stylesheet is going to bite you in the future.